env-files/account.template.env

```diff
@@ -33,6 +33,10 @@ WEBHOOK_BASE_URL=http://host.docker.internal:3001/webhooks/account-service
 REDIS_URL=redis://0.0.0.0:6379
 #REDIS_URL=redis://redis:6379                        #docker dev
 
+# Additional Redis client options as a JSON object string
+# See https://github.com/redis/ioredis/blob/2ed6414339bdda3e427ed1e8484263cc4848f2ac/lib/redis/RedisOptions.ts
+REDIS_OPTIONS='{"commandTimeout":5000}'
```

Comment:
I think we want to put the commandTimeout in the code defaults. That way if someone just wants to set some other option, they don't lose the default commandTimeout

Reply:
Ah I see. This is example and it is also set in `DEFAULT_REDIS_OPTIONS`. I missed that. No trouble here then.